ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -80,6 +80,10 @@ conformance_test_tool_args = [
         "CommandDeduplicationPeriodValidationIT:OffsetPruned",  # requires pruning not available in canton community
         "ActiveContractsServiceIT:AcsBeforePruningOffsetIsDisallowed",  # requires pruning not available in canton community
         "ActiveContractsServiceIT:AcsAtPruningOffsetIsAllowed",  # requires pruning not available in canton community
+        # TODO https://github.com/digital-asset/daml/issues/14199
+        #  reactivate the two following test as soon canton use an updated snapshot
```

Comment:
Given this PR should be fixing #14199, can't we remove the above comment now and ensure these ITs are enabled?